main.rb

```diff
@@ -0,0 +1,35 @@
+require 'sinatra'
+require_relative 'lib/drama/sentences'
+
+helpers do
+  def generate_sentence
+    # Splitting into an array is required for easily preventing duplicate vals
+    sent = Drama::Sentences.sentence.split(/\s/)
+    sent.each do |w|
+      num = sent.index(w)
+      w = w.gsub('[things]', Drama::Sentences.thing)
+      w = w.gsub('[sites]', Drama::Sentences.site)
+      w = w.gsub('[people]', Drama::Sentences.person)
+      w = w.gsub('[says]', Drama::Sentences.says)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.